### ISSUE TITLE
Nice Hash Balance

### ISSUE THREAD

**Issue by jeremyhammon** [RainbowMiner/RainbowMiner#1430]
Nice Hash balance reflects only the pending balance and not what is in your wallet.  I checked API keys and they were good so dug deeper.

Issue 1:
Because of the method being used to retrieve the wallet balance, the parameters @{"extendedResponse"="true"} need to be added to the Invoke-NHRequest. 

This alone will "fix" the wallet balance, but you are then unable to see the pending balance as the logic in the code currently pretty much says If you can get balance, do this, otherwise check the "rig2" values.  The issue with this is that the wallet balance is in one location and the pending is in a the rig location.  I modified the code (find it [here](https://github.com/jeremyhammon/RainbowMiner/commit/13edc34ccde91da391bfce7ec455119913beed2f)) but for some reason when doing this the "Balance ends up being the same as the total when displayed in the interface even though the values being used seem to be the correct ones.  I am trying to currently figure out if the code somewhere is duplicating the total value and using it for the balance value when displaying the information in the application or somewhere else that it is using it.

**Comment by jeremyhammon** [comment]
Looking at it some more.  It looks to be correct in the web interface with my changes.  The discrepancy I am seeing is in the console.  I will let it run for a little bit and see what happens.

**Comment by RainbowMiner** [comment]
Hi, thank you for looking into it. I think, the main discrepancy is between NH dedicated mining wallet and NH any wallet. I will double-check.

**Comment by jeremyhammon** [comment]
Yes, that is likely at least part of the reason.  I see 2 ways to approach it.  Either:
- Try to create logic to determine the case you are dealing with and have 2 separate logic trees to accommodate.
- Handle the NH wallet as a regular wallet except use the NH api to get the balance instead of a blockchain explorer.

**Comment by UselessGuru** [comment]
> but you are then unable to see the pending balance 

NiceHash has temporarly suspended this information in the API.
https://www.nicehash.com/blog/post/nicehash-maintenance-mode-explanation

**Comment by RainbowMiner** [comment]
Ah! That explains it. Thanks for letting me know.

**Comment by mcangeli** [comment]
This is still an issue. 
The NH balance only shows the "pending". Does anyone know if they have restored this info in the API?
